Project.toml

```diff
@@ -11,7 +11,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Requires = "0.5"
-Tracker = "0.2.2"
+Tracker = "0.2.2, 0.2.3, 0.2.4, 0.2.5"
```

Comment:
This is not required.
`0.2.2` means "Anything greater than or equal to`0.2.2` and less than `0.3.0`

Reply:
sorry, now fixed

Reply:
no worries